lib/util/cache/package/decorator.ts

```diff
@@ -76,7 +76,8 @@ export function cache<T>({
       finalKey
     );
 
-    const softTtl = ttlMinutes;
+    const ttlOverride = getTtlOverride(namespace);
```

Comment:
```suggestion
    const ttlOverride = getTtlOverride(finalNamespace);
```
so we ensure always string?